index.html

```diff
@@ -258,6 +258,56 @@ <h2>Myself</h2>
             </ul>
           </div>
         </div>
+
+        <footer>
+          <form
+            action="https://formspree.io/f/mgedkqkj"
+            method="post"
+            class="contact-form"
+            autocomplete="off"
+          >
+            <h2>Contact Me</h2>
+            <div class="parag">
+              <p>
+                If you have an application you are interested in developing, a
+                feature that you need built or a project that needs coding.I'd
+                love to help with it.
+              </p>
+            </div>
```

Comment:
- [ ] only the `<input>` fields should be inside the `<form>` tags, the `<h2>` and `<p>` should be outside of it.